model-optimizer/mo/utils/ir_reader/layer_to_class.py

```diff
@@ -125,8 +125,10 @@ def restore_correct_ports(graph: Graph):
         if 'out' in d:
             node = Node(graph, u)
             num_of_in_nodes = len(node.in_nodes())
-            # we need to check operation type, if it is const op, we don't renumber out ports
-            decremented_number = d['out'] - num_of_in_nodes if node.type != 'Const' else d['out']
+            decremented_number = d['out'] - num_of_in_nodes
```

Comment:
And looks like we can remove all this code related to renumbering for `Const` operations because we should keep the output port id as it was originally. So we need to revert and the previous changes done in this file.

Reply:
If output port id==1 then we cannot keep them, because Const op with one output port with id==1 will be inconsistent and we need to decrement id number. In other hand, If output port id==0 then we don't need to decrement it,  because  Const op with one output port with id==-1 will be inconsistent too. There are two different cases and we need to handle them. So I don't understand why we can remove these changes.